cmake/pcl_ccache.cmake

```diff
@@ -0,0 +1,31 @@
+# Needs CMake 3.4 at least
+# Inspired from:
+# * https://crascit.com/2016/04/09/using-ccache-with-cmake/
+# * https://stackoverflow.com/a/36515503/
+
+find_program(CCACHE_PROGRAM ccache)
+if(CCACHE_PROGRAM AND NOT PCL_DISABLE_CCACHE)
```

Comment:
If `ccache` is not enabled via the CMake option, we should not search for the program.